crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -2287,6 +2293,38 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn does_not_duplicate_global_registration_2311() {
+        let tmp_dir = TempTedgeDir::new();
+        let (mut converter, _http_proxy) = create_c8y_converter(&tmp_dir).await;
+
+        let measurement_message = Message::new(
+            &Topic::new_unchecked("te/device/main/service/my_measurement_service/m/my_type"),
+            r#"{"temperature": 21.37}"#,
+        );
+
+        // when auto registrating, local and cloud registration messages should be produced
```

Comment:
```suggestion
        // when auto-registered, local and cloud registration messages should be produced
```
Just fix that to avoid this annoying warning from Github actions in future.

Reply:
Fixed in cd49504